native/src/modules/app/StoreActionType.js

```diff
@@ -122,7 +122,7 @@ export type FetchCategoryFailedActionType = {|
   |}
 |}
 export type PushCategoryActionType = {|
-  type: 'PUSH_CATEGORY',
+  type: 'PUSH_CATEGORY' | 'REFRESH_CATEGORY',
```

Comment:
I think you should add here a separate action. which uses the same params. You can create a type: `CategoryParamsType` and use it in the `PushCategoryActionType` and `RefreshCategoryActionType`